envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/ssl/EnvoySANValidationTest.kt

```diff
@@ -0,0 +1,74 @@
+package pl.allegro.tech.servicemesh.envoycontrol.ssl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import pl.allegro.tech.servicemesh.envoycontrol.config.Ads
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlRunnerTestApp
+import pl.allegro.tech.servicemesh.envoycontrol.config.EnvoyControlTestConfiguration
+import pl.allegro.tech.servicemesh.envoycontrol.config.Xds
+
+class AdsEnvoySANValidationTest : EnvoySANValidationTest() {
+
+    companion object {
+        private val properties = mapOf(
+                "envoy-control.envoy.snapshot.trustedCaFile" to "/usr/local/share/ca-certificates/root-ca.crt"
```

Comment:
Do we even need `update-ca-certificates` if we set a path to our CA certificate directly anyway?

Reply:
We need it if we want to add a self signed cert to trusted certificates. Right now you're right I can remove it.